target/bin/delmailuser

```diff
@@ -86,81 +86,77 @@ then
   fi
 fi
 
-(
-  flock -e 200
-
-  for EMAIL in "${@}"
-  do
-    ERROR=false
+create_lock "$(basename "$0")"
+for EMAIL in "${@}"
+do
+  ERROR=false
 
-    # very simple plausibility check
-    [[ ${EMAIL} != *@*.* ]] && errex "No valid address: ${EMAIL}"
+  # very simple plausibility check
+  [[ ${EMAIL} != *@*.* ]] && errex "No valid address: ${EMAIL}"
```

Comment:
`errex "No valid email address: ${EMAIL}"`

Reply:
@casperklein Any idea why this is happening? I do not have the time to test this ATM. Looks pretty normal to me :)

Reply:
What do you mean? My suggestion was, to add the word "email" to the error message above.

Reply:
😆 I misread your comment. I thought this was an error message you‘re seeing. Nevermind my comment above.